package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "build": "rimraf dist && rollup -c",
     "watch": "rollup -c --watch",
-    "prepublish": "yarn build"
+    "prepublish": "in-publish && yarn build || not-in-publish"
```

Comment:
see https://www.npmjs.com/package/in-publish

Reply:
nice